geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/Z3IdxStrategy.scala

```diff
@@ -204,7 +204,9 @@ object Z3IdxStrategy extends StrategyProvider {
    * Eventually cost will be computed based on dynamic metadata and the query.
    */
   override def getCost(filter: QueryFilter, sft: SimpleFeatureType, hints: StrategyHints) =
-    if (filter.primary.length > 1) 200 else 400
+  // TODO check date range and use z2 instead if too big
+  // TODO also if very small bbox, z2 has ~10 more bits of lat/lon info
```

Comment:
do we have any check elsewhere for very small bbox? hopefully our map clicks aren't that small?